src/maps/deployment.ts

```diff
@@ -3,11 +3,13 @@ import { DeploymentFlowRunCreate, DeploymentFlowRunRequest, DeploymentUpdate, De
 import { DeploymentResponse } from '@/models/api/DeploymentResponse'
 import { Deployment } from '@/models/Deployment'
 import { createObjectLevelCan } from '@/models/ObjectLevelCan'
+import { SchemaResponseV2, schemaV2Mapper } from '@/schemas'
 import { MapFunction } from '@/services/Mapper'
+import { Schema } from '@/types/schemas'
 
 export const mapDeploymentResponseToDeployment: MapFunction<DeploymentResponse, Deployment> = function(source) {
   const rawSchema = source.parameter_openapi_schema ?? {}
-  const schema = this.map('SchemaResponse', rawSchema, 'Schema')
+  const schema = this.map('SchemaResponse', rawSchema as Schema, 'Schema')
```

Comment:
Why is this being cast here?

Reply:
I made the response type a union of the old response type and the new response type. Quick hack so that when I'm back cleaning up this model I can just remove the old type and the casting. I thought about writing type guards for that seemed overly complex for this. Open to other ideas though. This is admittedly a little hacky. But since we're just telling typescript what the response type is I don't think it matters much if we continue just telling it here.

Reply:
Fixed the type being used here. TY!